packages/sdk/src/interfaces/text-to-image.ts

```diff
@@ -0,0 +1,80 @@
+import { InterfaceDeclaration } from '../integration/definition'
+import z from '../zui'
+
+const ImageModelRefSchema = z.object({
+  id: z.string(),
+})
+
+const ImageModelSchema = z.object({
+  id: z.string(),
```

Comment:
**suggestion:**
```ts
const ImageModelSchema = ImageModelRefSchema.extend({ ... })
```

Reply:
Done in https://github.com/botpress/botpress/pull/13200/commits/4818294e6ce5315bfb3b97debb38bb794808ced5